src/coprocessor/metrics.rs

```diff
@@ -46,27 +46,27 @@ lazy_static! {
     pub static ref COPR_SCAN_KEYS: HistogramVec = register_histogram_vec!(
         "tikv_coprocessor_scan_keys",
         "Bucketed histogram of coprocessor per request scan keys",
-        &["req"],
+        &["req", "prio"],
         exponential_buckets(1.0, 2.0, 20).unwrap()
     ).unwrap();
     pub static ref COPR_SCAN_DETAILS: IntCounterVec = register_int_counter_vec!(
         "tikv_coprocessor_scan_details",
         "Bucketed counter of coprocessor scan details for each CF",
-        &["req", "cf", "tag"]
+        &["req", "cf", "tag", "prio"]
     ).unwrap();
     pub static ref COPR_ROCKSDB_PERF_COUNTER: IntCounterVec = register_int_counter_vec!(
         "tikv_coprocessor_rocksdb_perf",
         "Total number of RocksDB internal operations from PerfContext",
-        &["req", "metric"]
+        &["req", "metric", "prio"]
     ).unwrap();
     pub static ref COPR_EXECUTOR_COUNT: IntCounterVec = register_int_counter_vec!(
         "tikv_coprocessor_executor_count",
         "Total number of each executor",
-        &["type"]
+        &["type", "prio"]
     ).unwrap();
     pub static ref COPR_GET_OR_SCAN_COUNT: IntCounterVec = register_int_counter_vec!(
         "tikv_coprocessor_get_or_scan_count",
         "Total number of rocksdb query of get or scan count",
-        &["type"]
+        &["type", "prio"]
```

Comment:
let's call it "priority" to align with other metrics.

Reply:
What about those counters labeled with req?
Maybe use priority for the two metrics that use full spelling?

Reply:
@sunxiaoguang I think they can be left untouched. It's just we have existing metrics using "priority" as the label instead of an abbr. So using "priority" here should be more friendly and consistent.

Reply:
Sure

Reply:
It's done.